src/reducers/screen.ts

```diff
@@ -75,10 +75,13 @@ export function screen(state = initialState, action: AnyAction): ScreenState {
         }
         case types.selectView: {
             const newBcs: Record<string, BcMetaState> = {}
-            Array
-                .from(
-                    new Set(action.payload.widgets.map(widget => widget.bcName)) // БК которые есть на вьюхе
-                )
+            const bcNameList = action.payload.widgets.map(widget => {
+                if (bcNameList && bcNameList.find(widget.bcName as any)) {
```

Comment:
widget.bcName Is definetly not `any`, it is a string.  
I suspect typescript is correct on this one, as [Array.prototype.find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) takes a callback function as first argument, not the value